resources/views/patterns/02-organisms/sections/related-pages.blade.php

```diff
@@ -110,29 +111,61 @@
 
           if ($thumb_id) {
             $alt = get_post_meta($thumb_id, '_wp_attachment_image_alt', true);
+            if (!empty($thumb_crop)){
```

Comment:
What do you think, if we rewrite it like this:

```suggestion
           $thumb_size = 'thumbnail';
           if (!empty($thumb_crop)){
             if ($thumb_crop == 'landscape') $thumb_size = 'horiz__16x9';
             if ($thumb_crop == 'portrait') $thumb_size = 'vert__3x4';
           }
```

I think, that this code the same readable, and we will not use more different options than 'landscape' and 'portrait'.


Thanks,
Best Regards!

Reply:
Yeah, that's fine with me. I used switch case originally because I thought I might use a different size for the circular option, but since that uses the thumbnail size too this code is definitely more concise.